.evergreen/config.yml

```diff
@@ -2334,3 +2435,14 @@ buildvariants:
     display_name: "KMS KMIP ${os-ssl-40}"
     tasks:
       - name: ".kms-kmip"
+
+  - name: testgcpkms-variant
+    display_name: "GCP KMS"
+    run_on:
+      - debian11-small
```

Comment:
Is there a reason to run on Debian instead of Ubuntu?

Reply:
This is to match the GCE instance. That is using [debian11-small](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/0e75b9ee5ac3952df51f782ff33de137933e89ed/.evergreen/csfle/gcpkms/create-and-setup-instance.sh#L16). But there is no particular reason that Debian was chosen over other Linux distributions.